### ISSUE TITLE
Cannot import name 'map_azure_error_to_cli_error' during installing extension aks-preview

### ISSUE THREAD

**Issue by seanslma** [Azure/azure-cli#22916]
## Describe the bug

**Command Name**
`az aks maintenanceconfiguration list`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
cannot import name 'map_azure_error_to_cli_error'
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 556, in execute
    self.commands_loader.load_arguments(command)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/__init__.py", line 517, in load_arguments
    self.command_table[command].load_arguments()  # this loads the arguments via reflection
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 313, in load_arguments
    super(AzCliCommand, self).load_arguments()
  File "/opt/az/lib/python3.6/site-packages/knack/commands.py", line 104, in load_arguments
    cmd_args = self.arguments_loader()
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/command_operation.py", line 125, in arguments_loader
    op = self.get_op_handler(self.op_path)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/command_operation.py", line 59, in get_op_handler
    handler = import_module(mod_to_import)
  File "/opt/az/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ma/.azure/cliextensions/aks-preview/azext_aks_preview/custom.py", line 105, in <module>
    from ._helpers import (
  File "/home/ma/.azure/cliextensions/aks-preview/azext_aks_preview/_helpers.py", line 7, in <module>
    from azure.cli.command_modules.acs._helpers import map_azure_error_to_cli_error
ImportError: cannot import name 'map_azure_error_to_cli_error'

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks maintenanceconfiguration list -g {} --cluster-name {} {} {} {} -g {} --cluster-name {}`
```
The command requires the extension aks-preview. Do you want to install it now? The command will continue to run after the extension is installed. (Y/n): y
Run 'az config set extension.use_dynamic_install=yes_without_prompt' to allow installing extensions without prompt.
The installed extension 'aks-preview' is in preview.
```

## Environment Summary
```
Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-debian-bullseye-sid, Ubuntu 20.04.3 LTS
Python 3.6.10
Installer: DEB

azure-cli 2.32.0 *

Extensions:
aks-preview 0.5.70

Dependencies:
msal 1.16.0
azure-mgmt-resource 20.0.0
```

**Comment by yonzhan** [comment]
route to CXP team

**Comment by RakeshMohanMSFT** [comment]
@seanslma Thank you for reaching out, we are looking into it.

**Comment by RakeshMohanMSFT** [comment]
@seanslma can you please upgrade your azure CLI to its latest version [2.37.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)? i am able to get it to work with latest cli.

**Comment by seanslma** [comment]
I have no issues running the command on azure cloud shell (suppose azure shell has the latest version).

I tried to upgrade to the newest version on my machine but failed. Not sure this is due to azure or ubuntu.
```
This command is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
Your current Azure CLI version is 2.32.0. Latest version available is 2.37.0.
Please check the release notes first: https://docs.microsoft.com/cli/azure/release-notes-azure-cli
Do you want to continue? (Y/n):
...
Fetched 9212 kB in 7s (1252 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
azure-cli is already the newest version (2.32.0-1~focal).
The following packages were automatically installed and are no longer required:
  conntrack cri-tools ebtables kubernetes-cni socat
Use 'sudo apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 186 not upgraded.
CLI upgrade failed or aborted.
```

**Comment by seanslma** [comment]
Thanks Rakesh. 
Azure CLI upgraded successfully to the latest version 2.37.0 and there is no issue to run the command `az aks maintenanceconfiguration list`.
Please close this issue.
